### DISCUSSION TITLE
Testimonial not showing on profile

### DISCUSSION THREAD

**Discussion by AnkushSinghGandhi** [EddieHubCommunity/LinkFree#10347] [Q&A] [answered]
I am having issues with adding testimonials to my BioDrop profile 

https://github.com/EddieHubCommunity/BioDrop/tree/main/data/AnkushSinghGandhi/testimonials

I made my profile with forms and I am getting "No testimonial found"

<img width="1423" alt="Screenshot 2024-04-21 at 6 31 44 PM" src="https://github.com/EddieHubCommunity/BioDrop/assets/55637484/15409349-f319-47ce-89a6-fa279e6aa67a">

**Comment by eddiejaoude** [comment, marked as answer]
Testimonials will appear on your profile after the next deployment, as they are loaded via the json files - I hope that helps explain the situation

**Reply by AnkushSinghGandhi** [reply]
yeah sure got it
